portfolio/src/main/java/com/google/DataServlet.java

```diff
@@ -24,30 +24,34 @@
  
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data") 
-public class DataServlet extends HttpServlet {
-
-  private List<String> phrases;
+public class DataServlet extends HttpServlet 
+{ 
+  private List<String> comments;
 
    @Override
   public void init() {
-    phrases = new ArrayList<String>();
-    phrases.add("Writer");
-    phrases.add("Achiever");
-    phrases.add("Engineer");
-    phrases.add("Student");
-    phrases.add("Visionary");
+    comments = new ArrayList<String>();
   }
 
    @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String json = convertToJsonUsingGson(phrases);
+    String json = convertToJsonUsingGson(comments);
     response.setContentType("application/json;");
     response.getWriter().println(json);
   }
+
+   @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String message = request.getParameter("user-comment");
+    if(message == null || message.length() != 0){
+        comments.add(message);
```

Comment:
Do you want to add the `message` if its value is `null`?

Reply:
Oh I made a mistake with the conditional, I'll fix that